### PR TITLE
fix: staticImage should only set blur placeholder for jpeg,png,webp,avif

### PR DESCRIPTION
**Pull request by promer94**
should only use blur placeholder when file type is jpeg,png,webp,avif

### CODE REVIEW

File: packages/nextra/src/mdx-plugins/static-image.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { truthy } from '../utils'
 import { existsSync } from '../file-system'
 import { EXTERNAL_URL_REGEX, PUBLIC_DIR } from '../constants'
 
+const blurFormat = [".jpeg", ".png", ".webp", ".avif"]
```
Comment (shuding) [inline packages/nextra/src/mdx-plugins/static-image.ts:9]: Maybe `.jpg` too?

Comment (promer94) [reply packages/nextra/src/mdx-plugins/static-image.ts:9]: i got this extension array from here https://github.com/vercel/next.js/blob/6cfebfb02c2a52a1f99fca59a2eac2d704d053db/packages/next/client/image.tsx#LL702C82-L702C99

**Review by B2o5T — APPROVED** [review]

File: .changeset/forty-snakes-clean.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'nextra': patch
+'nextra-theme-blog': patch
+'nextra-theme-docs': patch
+---
+
+remove @react/skip-nev #1051 and fix static image
```
Comment (B2o5T) [inline .changeset/forty-snakes-clean.md:7]:
```suggestion
use blur placeholder when file type is `jpeg`, `png`, `webp`, `avif`, `jpg`
```
you don't remove @reachi/skip-nav

Comment (B2o5T) [reply .changeset/forty-snakes-clean.md]: Sorry 😅

**Pull request merged**